metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/ThreadServiceManager.java

```diff
@@ -37,17 +35,16 @@
  */
 @Getter
 @Slf4j
-// TODO: Move to Hive connector
+@Component
 public class ThreadServiceManager {
```

Comment:
Do we use this class in the main Application at all or can it be moved to Hive as the TODO said?

I know the spring context already has an `ExecutorService` so creating yet another one in here is redundanta nd not managed by spring (e.g. auto closed)